api/v1beta2/cloudstackmachine_types.go

```diff
@@ -139,10 +140,12 @@ type CloudStackMachineStatus struct {
 
 	// Status indicates the status of the provider resource.
 	// +optional
+	// +k8s:conversion-gen=false
 	Status *string `json:"status,omitempty"`
```

Comment:
curious why can't we convert the status?, is this a new field?

Reply:
yes, there are 3 new fields in v1beta2